microsoft-edge/webview2/release-notes/index.md

```diff
@@ -72,6 +72,338 @@ See also:
+   * `File`
+   * `Directory`
+
+* [CoreWebView2FileSystemHandlePermission Enum](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2filesystemhandlepermission?view=webview2-winrt-1.0.2651.64&preserve-view=true)
+   * `ReadOnly`
+   * `ReadWrite`
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2_23](/microsoft-edge/webview2/reference/win32/icorewebview2_23?view=webview2-1.0.2651.64&preserve-view=true)
+   * [ICoreWebView2_23::PostWebMessageAsJsonWithAdditionalObjects](/microsoft-edge/webview2/reference/win32/icorewebview2_23?view=webview2-1.0.2651.64&preserve-view=true#postwebmessageasjsonwithadditionalobjects)
+
+* [ICoreWebView2Environment14](/microsoft-edge/webview2/reference/win32/icorewebview2environment14?view=webview2-1.0.2651.64&preserve-view=true)
+   * [ICoreWebView2Environment14::CreateObjectCollection](/microsoft-edge/webview2/reference/win32/icorewebview2environment14?view=webview2-1.0.2651.64&preserve-view=true#createobjectcollection)<!--win32 only-->
+   * [ICoreWebView2Environment14::CreateWebFileSystemDirectoryHandle](/microsoft-edge/webview2/reference/win32/icorewebview2environment14?view=webview2-1.0.2651.64&preserve-view=true#createwebfilesystemdirectoryhandle)
+   * [ICoreWebView2Environment14::CreateWebFileSystemFileHandle](/microsoft-edge/webview2/reference/win32/icorewebview2environment14?view=webview2-1.0.2651.64&preserve-view=true#createwebfilesystemfilehandle)
+
+* [ICoreWebView2FileSystemHandle](/microsoft-edge/webview2/reference/win32/icorewebview2filesystemhandle?view=webview2-1.0.2651.64&preserve-view=true)
+   * [ICoreWebView2FileSystemHandle::get_Kind](/microsoft-edge/webview2/reference/win32/icorewebview2filesystemhandle?view=webview2-1.0.2651.64&preserve-view=true#get_kind)
+   * [ICoreWebView2FileSystemHandle::get_Path](/microsoft-edge/webview2/reference/win32/icorewebview2filesystemhandle?view=webview2-1.0.2651.64&preserve-view=true#get_path)
+   * [ICoreWebView2FileSystemHandle::get_Permission](/microsoft-edge/webview2/reference/win32/icorewebview2filesystemhandle?view=webview2-1.0.2651.64&preserve-view=true#get_permission)
+
+* [ICoreWebView2ObjectCollection](/microsoft-edge/webview2/reference/win32/icorewebview2objectcollection?view=webview2-1.0.2651.64&preserve-view=true)
+   * [ICoreWebView2ObjectCollection::get_Kind](/microsoft-edge/webview2/reference/win32/icorewebview2objectcollection?view=webview2-1.0.2651.64&preserve-view=true#get_kind)<!-- todo: test, b/c anchor section seems to not exist, in earlier release.  finding: get or get_Kind is missing from API Ref open PR: https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2objectcollection?view=webview2-win32-preview&branch=pr-en-us-94 - Also, get or get_Kind is missing from released Ref docs: https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2objectcollection?view=webview2-1.0.2646-prerelease -->
```

Comment:
ICoreWebView2ObjectCollection::get_Kind is missing from the open API Ref PR in the ObjectCollection type topic (so this link is "404", ie, defaults to top of webpage b/c there's no anchor section) (2651 Release):
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2objectcollection?view=webview2-win32-preview&branch=pr-en-us-94

A "get" method is also missing from previous Ref docs.  The topic says that this ObjectCollection is "Used to **get**, remove and add objects at the specified index", but no 'get' method is listed in API Ref (live Ref docs or latest draft PR).  Yet in older RelNotes list & latest draft RelNotes list, there's a get_Kind method (assume emitted by the script), pointing to such an anchor section.

Also, based on the type's description, I'd expect get_Item, not get_Kind.

Live Ref docs (2646-pre) for the type, missing a get method: 
https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2objectcollection?view=webview2-1.0.2646-prerelease

@peiche-jessica @victorhuangwq

Reply:
removed bc "There shouldn't be a ICoreWebView2ObjectCollection::get_Kind, it's a bug in the script"